atlas/packages/app/src/views/VideoView/VideoView.style.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from "@emotion/styled";
+import { ChannelAvatar, theme, VideoPlayer, VideoPreview } from "@joystream/components";
+
+export const Container = styled.div`
+	display: flex;
+	flex-direction: column;
+`;
+
+export const StyledVideoPlayer = styled(VideoPlayer)`
+	width: 100%;
+	height: 700px;
+`;
+
+export const InfoContainer = styled.div`
+	padding: ${theme.spacing.xxl};
+`;
+
+export const TitleActionsContainer = styled.div`
+	display: flex;
+	justify-content: space-between;
+`;
+
+export const Title = styled.h2`
+	font-size: ${theme.typography.sizes.h2};
+	margin: 0;
+`;
+
+export const ActionsContainer = styled.div``;
+
+export const Meta = styled.span`
+	display: block;
+	margin-top: ${theme.spacing.xxs};
+	color: ${theme.colors.gray[300]};
+`;
+
+export const StyledChannelAvatar = styled(ChannelAvatar)`
+	margin-top: ${theme.spacing.m};
+`;
+
+export const DescriptionContainer = styled.div`
+	margin-top: ${theme.spacing.xl};
+	border-top: 1px solid ${theme.colors.gray[800]};
+
+	p {
+		color: ${theme.colors.gray[300]};
+		line-height: 175%;
+		margin-top: ${theme.spacing.m};
+	}
+`;
+
+export const MoreVideosContainer = styled.div`
+	margin-top: 88px;
+`;
+
+export const MoreVideosHeader = styled.h5`
+	margin: 0 0 ${theme.spacing.m};
+	font-size: ${theme.typography.sizes.h5};
+`;
+
+export const MoreVideosGrid = styled.div`
+	display: grid;
+	grid-template-columns: repeat(auto-fit, minmax(320px, 1fr));
+	grid-gap: ${theme.spacing.xl};
+`;
+
+export const MoreVideosPreview = styled(VideoPreview)`
```

Comment:
Not sure if intentional, but on the `VideoView` page I see no hover effect on the grid of video previews, while I see them on the Carousels on the landing page.

Reply:
That's because there is no `onClick` handler set for those, so the hover animations is disabled. Once we add the handler, the animation should work properly.